DICOM/src/Microsoft.Health.DeID.SharedLib/Models/AgeType.cs

```diff
@@ -0,0 +1,15 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Dicom.DeID.SharedLib.Models
```

Comment:
When you call it sharedLib, maybe you want a shared anonymization library for FHIR, Dicom, and other healthcare data? Then I would suggest renaming name space to Microsoft.Health.Anonymization.Models